classes/person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+att_accessor :name, :age
+att_reader :id
+
+class Person < Nameable
```

Comment:
- [ ] You still haven't fixed this issue highlighted by the previous reviewer. You should use `attr_accessor` and `attr_reader`, not `att_accessor` and `att_reader`.

- [ ] The getter and setter should be within the class, not out of it.

  ```
  class Person < Nameable
    attr_accessor :name, :age
    attr_reader :id
    ...
  ```

Reply:
Hi, @TedLivist fixed all typos, person..rb, and capitalize_decorator class as requested.